package.json

```diff
@@ -9,7 +9,10 @@
   ],
   "author": "SAP SE (https://www.sap.com)",
   "license": "Apache-2.0",
-  "repository": "cap-js/graphql",
+  "repository": {
```

Comment:
FYI, the `"repository": "cap-js/graphql"` syntax is a shorthand that implies that the given repository is hosted on github.com. The fact that it works correctly can be seen by the repository link on the npm page: https://www.npmjs.com/package/@cap-js/graphql.

(see here for more details: https://docs.npmjs.com/cli/v11/configuring-npm/package-json#repository)

Reply:
hm, then I wonder why `npm publish` reports this as a warning.

Reply:
Hmm, that is strange